CI/file_check.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+FILENAME=$1
+
+DEST="\$(DESTDIR)"
+
+while IFS= read -r LINE_VAL; do
+
+	LINE=$(echo $LINE_VAL | xargs)	
+	IFS=' '
+	read -a COMMAND_ARGUMENTS <<< $LINE
+	LENGTH=${#COMMAND_ARGUMENTS[@]}
+
+	case $LINE in
+		
+		"DESTDIR="*)
+			IFS='='
```

Comment:
Some cases have an empty line between case and first line of code, some cases don't:
examples on lines 16-17:
``` "DESTDIR="*)
	IFS='='
``` 
and lines 21-23:
```
"install -d"*)  #creates directory from the PATH given as agrument

	if [[ ! ${COMMAND_ARGUMENTS[$LENGTH-1]} == *"(DESTDIR)"* ]]; then
```
I would go with all cases without empty line between case and first line of code.
The same recommendation  for IF (right after "if <condition>; then" to not have an empty line) and ELSE (before and after "else" to not have an empty line).

Indentation is enough in this case for code clarity.